src/main/java/com/github/awwkoala/selenium/page/CheckboxesPage.java

```diff
@@ -0,0 +1,59 @@
+package com.github.awwkoala.selenium.page;
+
+import org.assertj.core.api.Assertions;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class CheckboxesPage extends Page {
+
+  @FindBy(css = "input[type='checkbox']")
+  private List<WebElement> checkboxes;
+
+  public CheckboxesPage(WebDriver driver) {
+    super(driver);
+  }
+
+  public void goToCheckboxesPage() {
+    driver.get("http://theinternet.przyklady.javastart.pl/checkboxes");
+  }
+
+  public void clickCheckbox1() {
+    checkboxes.get(0).click();
```

Comment:
What about IndexOutOfBoundException? As you can see `clickCheckbox1` and `clickCheckbox2` are similar. You could create private method which takes an index argument and click. 

You can can rename it to `clickFirstCheckbox` `clickSecondCheckbox` there are looks more intuitive (for me of course ;))